### ISSUE TITLE
[mysql-async] [ERROR] ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client

### ISSUE THREAD

**Issue by giannistsitsis97** [brouznouf/fivem-mysql-async#165]
**[mysql-async] [ERROR] ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client**
on a vps with ubuntu 20.04 LTS , mysql 8.0

**Comment by XeNoMoRpH** [comment]
If I recall, MySQL changed how the passwords are stored by default in version 8. The usual solution is enabling `mysql_native_password`. You will either have to update the database user to use this native plugin or create a new user which does.

**Comment by XeNoMoRpH** [comment]
You should also see a warning when starting your server that says you should use MySQL 5.0 or MariaDB, I assume due to this reason.

**Comment by benzon** [comment]
And mysql-async dos not recommend using mysql 8.0 in any way it's to slow.

**Comment by XeNoMoRpH** [comment]
Just playing devil's advocate here. Can you point to where mysql-async says it does not recommend MySQL 8? I see some benchmarks [here](https://brouznouf.github.io/fivem-mysql-async/) which lists 8 as the slowest. Besides that, I don't see any specifics about using MySQL 8.

If it is the case that the author specifically does not recommend it, that should definitely be in the README for this repository.

**Comment by benzon** [comment]
It might be on the discord i just know he said that mysql 8.0 was not recommended, since the mysql lib used was not optimized for it and performed very slowly just like the benchmarks.

**Comment by XeNoMoRpH** [comment]
You are right that when starting up the server, it mentions not using MySQL 8. I tried MariaDB 10.6, but it was overall slower in my limited test.

**Comment by benzon** [comment]
i would still default to MariaDB10.4 at this time, or MySQL 5.7 at the current time
